android/src/test/java/com/fidelreactlibrary/adapters/FidelVerificationSetupAdapterTest.java

```diff
@@ -0,0 +1,28 @@
+package com.fidelreactlibrary.adapters;
+
+import com.fidelreactlibrary.fakes.ReadableMapStub;
+import com.fidelapi.entities.CardVerificationConfiguration;
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class FidelVerificationSetupAdapterTest {
+
+    private FidelVerificationSetupAdapter sut = new FidelVerificationSetupAdapter();
+
+    @Test
+    public void test_ShouldMapEmptyMapCorrectly() {
+        CardVerificationConfiguration cardConfig = sut.map(ReadableMapStub.mapWithNoKey());
+        assertEquals("", cardConfig.id);
+        assertEquals("", cardConfig.consentId);
+        assertEquals("", cardConfig.last4Digits);
+    }
+
+    @Test
+    public void test_ShouldMapNonEmptyValidMapCorrectly() {
+        CardVerificationConfiguration cardConfig = sut.map(ReadableMapStub.mapWithVerificationSetup());
+        assertEquals("123", cardConfig.id);
```

Comment:
For the successful mapping tests, I would suggest having a test for each of the assertion. Also, I would also add a few more tests, since they are possible in production:
1. A map without one of the required fields (id, consent ID). As JS is not strong-typed, they can be omitted. The tests should describe what happens, in that scenario.
2. A map without the optional field `last4Digits`.